src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -56,10 +59,24 @@ const webhookRouter = (context: WebhookContext) => {
 				GithubWebhookMiddleware(issueWebhookHandler)(context);
 			}
 			break;
+		case "pull_request":
+			if (context.action && VALID_PULL_REQUEST_ACTIONS.includes(context.action)) {
+				GithubWebhookMiddleware(pullRequestWebhookHandler)(context);
+			}
+			break;
+		case "pull_request_review":
+			GithubWebhookMiddleware(pullRequestWebhookHandler)(context);
+			break;
+		case "create":
+			GithubWebhookMiddleware(createBranchWebhookHandler)(context);
+			break;
+		case "delete":
+			GithubWebhookMiddleware(deleteBranchWebhookHandler)(context);
+			break;
 	}
 };
 
-const createHash = (data: BinaryLike, secret: string): string => {
+export const createHash = (data: BinaryLike, secret: string): string => {
```

Comment:
Personally I wouldn't export this `createHash`, but duplicate it in the test file.

Reply:
We want a valid test, so if we have a bug in the `createHash` function, like `const createHash = () => { return 'always_success'`, the test will still "success", but there's a bug, so test is not a valid test.

---------

Instead, we could just duplicate the method in the test, as that `sha256` is an intention we want to test.